language/move-lang/src/shared/mod.rs

```diff
@@ -58,8 +58,9 @@ impl Address {
         assert!(result.len() >= 32);
```

Comment:
I couldn't help but notice this actually. Not needed now, but might be a good idea to come back and use `ADDRESS_LENGTH` for these constants as well.

Reply:
Good point. I'll make a separate PR for that.

Reply:
See https://github.com/libra/libra/pull/2534